### PR TITLE
feat: use official sdk package for nova and rename packages

### LINKED ISSUE #868: [Task]: Rename "@iota/sdk" to "@iota/sdk-stardust"
### Task description

To make imports more organized with the SDK, we should rename the current use of the stardust version of "@iota/sdk" to "@iota/sdk-stardust"

### Requirements

TBD

### Acceptance criteria

TBD

### Creation checklist

- [ ] I have assigned this task to the correct people
- [ ] I have added the most appropriate labels
- [ ] I have linked the correct milestone and/or project

### LINKED ISSUE #1326: [Task]: Add sdk dependency to use official npm package
### Task description

- [ ] Update dependency to use https://github.com/iotaledger/iota-sdk/releases/tag/iota-sdk-nodejs-v2.0.0-alpha.1
- [ ] Update CI to remove sdk build
- [ ] Cleanup es-lint ignores

### PR DESCRIPTION
**Pull request by VmMad**
# Description of change

Fixes #1326 
Fixes #868 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

1. PR checks
2. Local build

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/nova-build-temp.yaml [hunk 1]
```diff
@@ -1,129 +1,54 @@
...
-        required: false
-        default: "7e2b3547851280a310735861241ab3e0d239aa5a"
-      environment:
-        type: choice
-        description: "Select the environment to deploy to"
-        options:
-          - nova
-        required: false
-        default: nova
       deployment:
```
Comment (begonaalvarezd) [inline .github/workflows/nova-build-temp.yaml:6]: I guess we dont need this anymore

File: .github/workflows/nova-build-temp.yaml [hunk 2]
```diff
@@ -1,129 +1,54 @@
...
 
+env:
+  DEPLOY_ENV: nova
+
 jobs:
-  build-and-deploy:
+  set_env_vars:
+    name: Set Environment Variables
     runs-on: ubuntu-latest
-    environment: release
```
Comment (begonaalvarezd) [inline .github/workflows/nova-build-temp.yaml:26]: this cant be removed

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
We should also cleanup `setup_nova.sh` right?

**Comment by VmMad** [comment]
> We should also cleanup `setup_nova.sh` right?

@begonaalvarezd, @msarcev asked us to not drop the script yet, in case we need it

File: client/src/app/components/footer/Footer.scss [hunk 3]
```diff
@@ -4,15 +4,46 @@
 @import "../../../scss/variables";
 
 footer {
+    ul li {
```
Comment (begonaalvarezd) [inline client/src/app/components/footer/Footer.scss:7]: revert and ignore all css changes in this PR

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**